pkg/controllers/csi/provisioner/controller.go

```diff
@@ -222,9 +233,8 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(
 			log.Info("error when updating agent from image", "error", err.Error())
 			// reporting error but not returning it to avoid immediate requeue and subsequently calling the API every few seconds
 			return true, nil
-		} else if updatedDigest != "" {
-			dynakubeMetadata.LatestVersion = ""
-			dynakubeMetadata.ImageDigest = updatedDigest
+		} else {
```

Comment:
I don't think the `else` is not necessary, maybe `updatedDigest != ""`?

Reply:
Fixed
else clause is needed because of linter complains :smiley_cat:

Reply:
Fixed